lua/neo-tree/git/init.lua

```diff
@@ -0,0 +1,176 @@
+local Path = require("plenary.path")
+local utils = require("neo-tree.utils")
+
+local os_sep = Path.path.sep
+
+local M = {}
+
+local function execute_command(cmd)
+  local result = vim.fn.systemlist(cmd)
+
```

Comment:
This should fix the bug with a clean git status:
```suggestion
  if #result == 0 then
    return true, {}
  end

```